parlai/core/agents.py

```diff
@@ -392,6 +396,18 @@ def get_agent_module(dir_name):
         module_name = 'parlai.agents.legacy_agents.{m}.{m}_v{v}'.format(
             m=model_name, v=s[2])
         class_name = name_to_agent_class(model_name)
+    elif dir_name.startswith('projects:'):
+        # e.g. -m projects:personachat:kvmemnn
+        s = dir_name.split(':')
+        if len(s) != 3:
+            raise RuntimeError('projects paths should follow pattern '
+                               'projects:folder:model; you used {}'
+                               ''.format(dir_name))
```

Comment:
Maybe we should say that this maps to `parlai.projects.folder.model.model`?

Reply:
it doesn't, projects is outside of parlai